src/tabs/tabs.js

```diff
@@ -22,7 +24,7 @@ angular.module('mm.foundation.tabs', [])
 
   ctrl.addTab = function addTab(tab) {
     tabs.push(tab);
-    if (tabs.length === 1 || tab.active) {
+    if ($scope.openFirstTab && tabs.length === 1 || tab.active) {
```

Comment:
I usually prefer fewer logical branches so I'd rather go with the previous version. :smiley: